### DISCUSSION TITLE
Optional Markdown doc comments

### DISCUSSION THREAD

**Discussion by hydroper** [dotnet/csharplang#7342] [Language Ideas]
Something similiar to the `Nullability` option in `csproj`, but for causing the C# compiler to interpret a different doc comment form in Markdown (different from the XML schema, but can be somehow compatible since Markdown supports HTML tags).

Remarks:

- Shortcut reference links: `[SomeLexicalName]`, `[n](SomeLexicalName)`, <code>[\`SomeLexicalName\`]</code>
- Backwards compatibility with XML schema through use of specialized HTML tags
- This won't break existing C# projects as this is optional

It'd look like this in .csproj:

```xml
<MarkdownDocs/>
```

**Comment by CyrusNajmabadi** [comment]
Dupe of https://github.com/dotnet/csharplang/discussions/891
